launcher/modManager/cmodlist.h

```diff
@@ -49,6 +49,8 @@ class CModEntry
 	bool isUpdateable() const;
 	// installed
 	bool isInstalled() const;
+	// vcmi essential filea
```

Comment:
useless comment. Lets remove

Reply:
comments for method above are about same:) so I just keep existing "style" :)

Reply:
```suggestion
	// vcmi essential files
```